src/30-secw-update-database-right.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+#  \brief   Make sure the security wallet db is created with security wallet user
+#  \author  Clement Perrette
+#
+
+SECW_USER="secw-daemon"
+SECW_GROUP="secw-daemon"
+
+SECW_DIR="/var/lib/fty/fty-security-wallet"
+SECW_DB="database.json"
+
+# Credential asset mapping is included into the security wallet daemon
+CAM_DB="mapping.json"
+
+die() {
+    echo "FATAL: $*" >&2
+    exit 1
+}
+
+getent passwd "${SECW_USER}" >/dev/null \
+|| die "The '${SECW_USER}' user account is not defined in this system"
+
+
+getent group "${SECW_GROUP}" >/dev/null \
+|| die "The '${SECW_GROUP}' group account is not defined in this system"
+
+chown ${SECW_USER}:${SECW_GROUP} "${SECW_DIR}"
+chmod 0771 "${SECW_DIR}"
+
+chown ${SECW_USER}:${SECW_GROUP} "${SECW_DIR}/${SECW_DB}"
```

Comment:
There is no check (or action) to make sure the `SECW_DB` and `CAM_DB` files exist; for example during a first run after installation or a factory reset they would not be there.

This would cause the scriptlet to return with a non-zero code and so fail the `ipc-meta-setup.service`.

If the files must be there (even if empty), `touch` or otherwise populate them (echo, copy, ...)
If it is acceptable that they do not exist (and the daemon later creates them), add an `if` for that.

Also, does it suffice that this scriptlet runs once in a lifetime of a deployment, or should it reinforce the rights "just in case" on every boot (if yes, rename to `*.everytime.sh`)?